src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -17,6 +17,22 @@ using OptionalWhisperGenerationConfig = std::optional<WhisperGenerationConfig>;
 
 using RawSpeechInput = std::vector<float>;
 
+struct OPENVINO_GENAI_EXPORTS WhisperDecodedResultChunk {
+    // start of chunk in seconds
+    float start_ts;
+
+    // end of chunk in seconds
+    // -1.0f if chunk started but model did not predict an ending timestamp
+    // can happen if audio is cut off in the middle of a word
+    float end_ts = -1.0f;
+    std::string text;
+};
+
+class OPENVINO_GENAI_EXPORTS WhisperDecodedResults : public DecodedResults {
```

Comment:
we don't need to export these classes because they don't have any methods, ctros.

Reply:
Done